packages/text-area/src/TextArea.js

```diff
@@ -1,45 +1,57 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
-import TextAreaPresenter from "./presenters/TextAreaPresenter";
+import Input from "@hig/input";
+
+function getWebLightBorders(props, themeData) {
+  const { isDisabled, hasFocus, hasHover } = props;
+
+  if (hasFocus) {
+    return themeData[`input.focus.borderBottomColor`];
+  }
+  if (hasHover && !isDisabled) {
+    return themeData[`input.hover.borderBottomColor`];
+  }
+  return themeData[`input.borderColor`];
+}
+
+function stylesheetOverride(stylesheet, props, themeData, theme) {
+  const borderStyles =
+    props.variant === `line`
+      ? `rgba(128, 128, 128, 0.2)`
+      : themeData[`input.borderColor`];
+  const textAreaWrapper =
+    theme === `hig-light`
+      ? {
+          ...stylesheet.wrapper,
+          borderColor: getWebLightBorders(props, themeData),
+          borderBottomColor: getWebLightBorders(props, themeData)
+        }
+      : {
+          ...stylesheet.wrapper,
+          borderLeftColor: borderStyles,
+          borderRightColor: borderStyles,
+          borderTopColor: borderStyles
+        };
+  const textArea = {
+    ...stylesheet.input,
+    display: `block`,
+    resize: `none`,
+    padding: `${themeData["input.verticalPadding"]}
+      ${themeData["input.boxType.horizontalPadding"]}`
+  };
+
+  return {
+    ...stylesheet,
+    wrapper: textAreaWrapper,
+    input: textArea
+  };
+}
+
+const variantTypes = ["line", "box"];
```

Comment:
Good call on breaking this out to a separate `stylesheet.js` type of file. I am reluctant to write test for the `web light` case since that is legacy and should go away sooner than later.